azure-client-runtime/src/main/java/com/microsoft/azure/util/ServiceFutureUtil.java

```diff
@@ -1,37 +0,0 @@
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- */
-
-package com.microsoft.azure.util;
-
-import com.microsoft.azure.OperationStatus;
-import com.microsoft.rest.ServiceCallback;
-import com.microsoft.rest.ServiceFuture;
-import rx.Observable;
-import rx.Single;
-import rx.functions.Func1;
-
-/**
- * Azure-specific helper methods for creating ServiceFuture instances.
- */
-public class ServiceFutureUtil {
-    /**
-     * Creates a ServiceFuture from an observable representing a long-running operation.
-     * @param observable The observable representing the long-running operation.
-     * @param callback The callback to run when the service future is completed.
-     * @param <T> The type of entity expected to be returned from the long-running operation.
-     * @return A ServiceFuture representing the long-running operation.
-     */
-    public static <T> ServiceFuture<T> fromLRO(Observable<OperationStatus<T>> observable, ServiceCallback<T> callback) {
```

Comment:
I wrote this for use in gen code for LROs-- implementing e.g. `create(ServiceCallback)` in terms of `beginCreate()`. Is there something you wanted that to migrate to?

Reply:
I was thinking that this Util class was only temporary until we removed the create(ServiceCallback) function. Since we're removing that function for v2 (right?), I figured that we might as well start removing it now. What kind of changes need to happen in gen code (branch v2) to not need this Util class?

Reply:
If we actually are getting rid of ServiceCallback in general then we don't need ServiceFutureUtil.fromLRO. So far I think we've just decided that ListOperationCallback is going away. I'd be open to killing ServiceCallback but I'm not sure I can make an informed decision about it.